src/QueryPressure.WinUI/Services/Language/ILanguageService.cs

```diff
@@ -0,0 +1,12 @@
+namespace QueryPressure.WinUI.Services.Language;
+
+public interface ILanguageService
+{
+  string GetCurrentLanguage();
+
+  IDictionary<string, string> GetStrings();
```

Comment:
Do we really need all the strings as a single object anywhere?

Reply:
> 

I do not know how fast is `_resourceManager.GetResources(...)` if we can execute it for each localized string separately than we do not need all strings.